frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1729,7 +1762,13 @@ function ReaderFooter:genAllFooterText()
         -- Skip empty generators, so they don't generate bogus separators
         local text = gen(self)
         if text and text ~= "" then
-            table.insert(info, BD.wrap(text))
+            if self.settings.item_prefix == "compact_items" then
+                -- remove whitespace from footer items if symbol_type is compact_items
+                text = text:gsub('%s', '')
+                table.insert(info, BD.wrap(text))
+            else
+                table.insert(info, BD.wrap(text))
```

Comment:
Put the `table.insert(info, BD.wrap(text))` outside the if/else as it's the same. And so, no `else `needed.

Reply:
Should I hold off on any more commits, or keep putting them in with each comment? :)

I notice a lot of builds happening ...

Reply:
It's better to the environment to bundle them, but they do auto-cancel.

Reply:
(I was a software designer in a prior life, but it's been 17 years. I'm rusty)

Reply:
Thanks. I'll hold off committing until there are a few. Please do let me know if I'm holding anything up.